tox.ini

```
@@ -14,6 +14,7 @@ python =
 
 [testenv]
 commands = ./runtests.py --nolint
+allowlist_externals=*
```

Not that I have a personal preference, but I suggest to keep the code style found around  
```suggestion
allowlist_externals = *
```